src/main/resources/application-test.properties

```diff
@@ -0,0 +1,4 @@
+server.port=8081
```

Comment:
Te propertiesy są do testów? Jeśli tak to nie powinny znajdować się w src.

Reply:
Mamy 3 configuracje:
- prod (file or memory)
- dev (file or memory)
- test (file or memory)
Przeniosłem teraz test to test/resources folder